app/views/profile/index.html.slim

```diff
@@ -3,19 +3,21 @@
 - title t('titles.profile')
 - btn_cls = 'btn btn-primary p0 w-60p bg-light-blue blue h6 regular border-box center'
 
-- if @recovery_code
+- if @view_model.recovery_code
```

Comment:
the *awesome* think about view models is that now you can put this logic in there instead of in the view. Basically, you'd call `@view_model.recovery_code_partial` here and then in the `recovery_code_partial` method you'd return either a partial with the content below or `shared/null` (nothing) of there is not recovery code.